src/appleseed/renderer/kernel/lighting/directlightingintegrator.cpp

```diff
@@ -233,15 +233,19 @@ void DirectLightingIntegrator::compute_outgoing_radiance_light_sampling_low_vari
     }
 
     // Add contributions from non-physical light sources only.
-    for (size_t i = 0, e = m_light_sampler.get_non_physical_light_count(); i < e; ++i)
+    if (m_light_sample_count > 0)
```

Comment:
> I've found that when light sample count < 0

I assume you mean `light sample count < 1`? Because `light sample count< 0` doesn't make sense and shouldn't be allowed from .studio.

The `light sample count < 1` case is a confirmed bug: https://github.com/appleseedhq/appleseed/issues/1467

I suggest you don't worry about it for now, we'll fix it at some point.